README.md

```diff
@@ -1,9 +1,12 @@
-<img src="https://www.ga4gh.org/wp-content/themes/ga4gh-theme/gfx/GA-logo-horizontal-tag-RGB.svg" alt="GA4GH Logo" style="width: 400px;"/>
+<img src="https://www.ga4gh.org/wp-content/themes/ga4gh/dist/assets/svg/logos/logo-full-color.svg" alt="GA4GH Logo" style="width: 400px;"/>
+
+![release_badge](https://img.shields.io/github/v/tag/ga4gh/task-execution-schemas)
+
 
 Task Execution Service (TES) API
 ======================================
-<sup>`master` branch status: </sup>[![Build Status](https://travis-ci.org/ga4gh/task-execution-schemas.svg?branch=master)](https://travis-ci.org/ga4gh/task-execution-schemas?branch=master)
-<a href="https://raw.githubusercontent.com/ga4gh/task-execution-schemas/master/openapi/task_execution_service.openapi.yaml"><img src="http://online.swagger.io/validator?url=https://raw.githubusercontent.com/ga4gh/task-execution-schemas/master/openapi/task_execution_service.openapi.yaml" alt="Open API Validator" height="20em" width="72em"></A>
```

Comment:
Kind of a pity to miss out on them, no?

I think for the build status, we could update the badge to use GitHub Actions instead of Travis.
For the OpenAPI validation, it still seems to work, but maybe we should replace `http` with `https`.

Also, instead of the HTML, we should just use markdown syntax - and move them above the header, together with the release badge.

While we are at it, we could also add a license badge.

Reply:
Good idea. Working on it.